client/controllers/incidentReports/incidentForm.coffee

```diff
@@ -70,10 +74,14 @@ Template.incidentForm.onRendered ->
 
   @$('#add-incident').validator()
   #Update the validator when Blaze adds incident type related inputs
-  @autorun ->
-    instance.incidentType.get()
-    Meteor.defer ->
-      instance.$('#add-incident').validator('update')
+  @autorun =>
+    @incidentType.get()
+    Meteor.defer =>
+      @$('#add-incident').validator('update')
+
+  if not @incidentData.locations.length
```

Comment:
Good call. How's [this for a solution](https://github.com/ecohealthalliance/eidr-connect/blob/ba0b45193343a2a8ef6a4ee0581863a525fe9f37/client/controllers/incidentReports/suggestedIncidentsModal.coffee#L219)?